apps/web/package.json

```diff
@@ -43,6 +44,8 @@
     "plyr-react": "^5.1.0",
     "react": "^18.2.0",
     "react-copy-to-clipboard": "^5.1.0",
+    "react-dnd": "^16.0.1",
+    "react-dnd-html5-backend": "^16.0.1",
```

Comment:
@harish-sethuraman are we ok with this lib? tagging you cuz u are expert in this 😉

Reply:
If we have vertical collection, sorting and searching then we can go with `react-beautiful-dnd` if that cannot handle what you want then react dnd is fine as well.